README.rst

```diff
@@ -35,6 +35,8 @@ be done easily using conda or pip::
   conda install cookiecutter
```

Comment:
it might be better to change this line to:

`conda install -c conda-forge cookiecutter gitpython`